templates/details/empty-docs.html

```diff
@@ -10,7 +10,7 @@ <h4>Add docs to your charm</h4>
         Looks like you haven’t added any docs yet. Docs help your users learn how to use your charms, and encourage them to use them the way you intended.
       </p>
       <p>
-        <a class="p-button--positive" href="#">Learn how to add docs to your charm</a>
+        <a class="p-button--positive" href="https://discourse.charmhub.io/t/how-to-add-documentation-to-your-charm/3784">Learn how to add docs to your charm</a>
```

Comment:
Why not link to the tutorial?

Reply:
the tutorials on the site will be linked once tutorials are live
For now the tutorials uploaded are meh. Just want to make sure that we are not sending to lost pages

Reply:
you know what actually I'll link to the tutorials. It's not too weird.